apps/web/src/components/Profile/components/LanguageProfile/Display.tsx

```diff
@@ -103,10 +103,10 @@ const Display = ({
         <FieldDisplay
           label={intl.formatMessage({
             defaultMessage:
-              "Second language level (Comprehension, Written, Verbal)",
-            id: "zF0F6w",
+              "Second language level (Reading, Writing, Oral interaction)",
```

Comment:
Change to **Second language level (reading, writing, oral interaction)** for  `defaultMessage` and `description`.